gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -0,0 +1,340 @@
+= Renewing Flex Gateway Registration
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Flex Gateway uses a managed PKI certificate to communicate with Anypoint Platform. This certificate is generated when you register Flex Gateway in either Connected Mode or Local Mode. Updating the certificate ensures the continued operation of your applications. You update a certificate by renewing registration, via the `flexctl registration renew` CLI.
+
+Attempting to run Flex Gateway with an expired certificate in Connected Mode results in the following:
+
+* New replicas fail to download API configurations from Anypoint Platform, thereby becoming unusable. Existing clusters fail to reload.
+* Logs and metrics fail to upload to Anypoint Platform. Troubleshooting using Anypoint Platform is not possible.
+* Metering information fails to upload to Anypoint Platform. MuleSoft is unable to collect or report usage metrics.
+
+Attempting to run Flex Gateway with an expired certificate in Local Mode results in the following:
+
+* Metering information fails to upload to Anypoint Platform. MuleSoft is unable to collect or report usage metrics.
+
+If a certificate expires, you can still invoke `flexctl registration renew`, which updates the expired certificate.
+
+To avoid disruptions:
+
+. <<before-you-begin,Download the latest version of Flex Gateway.>>
+. <<renewal-cli-overview,Renew your Flex Gateway registration.>>
+. <<inspection-cli-overview,Verify (inspect) the success of the renewal process.>>
+
+[[before-you-begin]]
+== Before You Begin
+
+* xref:flex-install.adoc[Download Flex Gateway].
++
+To invoke the registration renewal `flexctl` command, download the latest version of Flex Gateway. It is not necessary to register and run this version of the gateway.
+
+[[renewal-cli-overview]]
+== Registration Renewal
+
+Flex Gateway includes the following CLI command to renew registration:
+
+----
+flexctl registration renew
+----
+
+The command renews an existing Flex Gateway registration with Runtime Manager. For authentication, you specify *either* user credentials or connected appl credentials. Authentication with user credentials requires the `--username` and `--password` flags. Authentication with connected app credentials requires the `--client-id` and `--client-secret` flags.
+
+Invoke the following `help` command for information about usage:
```

Comment:
Maybe this is a separate task?

Reply:
I'm thinking that the "help" command isn't actually part of the task. It's more of a reference for something they can do, as opposed to something they need to do to renew their registration.

Reply:
What about moving the help bits to a separate topic at the end?

Reply:
Fixed!